multicall/constants.py

```diff
@@ -1,5 +1,12 @@
 from enum import IntEnum
 
+try:
+    from brownie import web3, network
+    if network.is_connected(): w3 = web3
```

Comment:
this should fall through to web3py if brownie network is not connected (not run from a brownie script)

Reply:
Fix'd